docs/release-notes/NuGet-5.8.md

```diff
@@ -0,0 +1,170 @@
+
+* Enable PM UI experiment: Package Recommender - [#9845](https://github.com/NuGet/Home/issues/9845)
+
+* `RepositoryMetadata.Equals()` does not account for Branch and Commit properties - [#9613](https://github.com/NuGet/Home/issues/9613)
+
+* In command line restore, with multi targeted projects, NuGet should read the target framework related information from the inner build - [#9869](https://github.com/NuGet/Home/issues/9869)
+
+* Read Runtime Identifier graph through TargetFrameworkInformation item - [#9874](https://github.com/NuGet/Home/issues/9874)
+
+* Static graph restore is inconsistent with regards to CrossTargeting property in compared to Visual Studio and regular MSBuild evaluation restore - [#9881](https://github.com/NuGet/Home/issues/9881)
+
+* In static graph restore, with multi targeted projects, NuGet should read the target framework related information from the inner build. - [#9870](https://github.com/NuGet/Home/issues/9870)
+
+* Allow `net5.0-platform` projects to be loaded and restored in Visual Studio - [#9863](https://github.com/NuGet/Home/issues/9863)
+
+* Display the resolved version in the PM UI - [#9826](https://github.com/NuGet/Home/issues/9826)
+
+* Solution Explorer not showing all NuGet package dependencies - [#9898](https://github.com/NuGet/Home/issues/9898)
+
+* Unnecessary assets file writes can lead to extra design time builds - [#9903](https://github.com/NuGet/Home/issues/9903)
+
+* [CPVM] Error on floating transitive dependencies defined in Directory.Packages.props - [#9384](https://github.com/NuGet/Home/issues/9384)
+
+* Update the SPDX license list - [#9946](https://github.com/NuGet/Home/issues/9946)
+
+* Error when a Project has CPVM enabled but there are not any PackageVersion items defined - [#9341](https://github.com/NuGet/Home/issues/9341)
+
+* VS 2019 crashes after opening Manage NuGet Packages: icon causes unhandled exception in image conversio - [#9696](https://github.com/NuGet/Home/issues/9696)
+
+* NuGet.Packaging.Extraction needs ilmerge to exclude Newtonsoft.Json - [#9966](https://github.com/NuGet/Home/issues/9966)
+
+* Packing with ContinuePackingAfterGeneratingNuspec=false should not fail when there are no errors - [#9786](https://github.com/NuGet/Home/issues/9786)
+
+* Clicking NU code in Visual Studio Error List window should go to https://docs.microsoft.com/nuget/reference/errors-and-warnings/ - [#9934](https://github.com/NuGet/Home/issues/9934)
+
+* Use 'https://' when adding new package source through Visual Studio options - [#9974](https://github.com/NuGet/Home/issues/9974)
+
+* [CPVM] Cannot import packages file from a custom directory - [#9841](https://github.com/NuGet/Home/issues/9841)
+
+* `RuntimeEnvironmentHelper.IsRunningOnVisualStudio` performance issue on Mono - [#9989](https://github.com/NuGet/Home/issues/9989)
+
+* Solution Load Restore runs after first 50 nominations instead of waiting for all nominations - [#9982](https://github.com/NuGet/Home/issues/9982)
```

Comment:
@chgill-MSFT There's 2-3 issues in the bugs section that have to do with performance on solution load, but no individual issue is worth enough to call out individually.